server/broadcast_test.go

```diff
@@ -0,0 +1,21 @@
+package server
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/livepeer/go-livepeer/core"
+)
+
+func TestStopSessionErrors(t *testing.T) {
+	errs := []string{
+		"Unable to read response body for segment 4 : unexpected EOF",
```

Comment:
Should we add an error message that should not stop a session and test that `shouldStopSession` returns false as expected?

Reply:
Fixed